crates/mun_abi/src/builtin.rs

```diff
@@ -0,0 +1,122 @@
+///! A module that defines information for built-in (or primitive) types.
+use crate::{Guid, HasStaticTypeId, TypeId};
+
+/// Defines functions for built-in types like f32, i32, etc.
+pub trait BuiltinType: HasStaticTypeId {
```

Comment:
Previously we referred to these as `Primitive` types. Are we transitioning to built-in?

Reply:
Good one! Both work for me, but for consistency Ill rename this one.